building-with-ocean/build-a-marketplace/customising-your-market.md

```diff
@@ -0,0 +1,184 @@
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-15.png)
+
+Additionally, the asset type current says dataset, and we need to change this so that it says photo. The simplest way to do this is to change the title of the asset type without changing anything else. Ocean can handle selling any digital asset that can be accessed via a URL, so no further changes are needed to accommodate selling photos.
+
+Open up src/components/Publish/Metadata/index.tsx and change line 28 so that it says ‘Photo’
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-18.png)
+
+Great, now our publish page explains that users should be publishing photos and photo is provided as an asset type option. We’ll also leave algorithm as an option in case some data scientists want to do some analysis or image transformation on the photos.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-19.png)
+
+There is one more thing that is fun to change before we move away from the publish form. You’ll notice that Ocean Market V4 now has a cool SVG generation feature that creates the images for the Data NFT. It creates a series of pink waves. Let’s change this so that it uses our brand colors in the waves!
+
+Open up /src/@utils/SvgWaves.ts and have a look at lines 27 to 30 where the colors are specified. Currently, the pink color is the one used in the svg generator. You can replace this with your own brand color:
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-21.png)
+
+If you’re interested in doing some further customization, take a look at lines 53 to 64. You can change these properties to alter how the image looks. Feel free to play around with it. We’ve increased the number of layers from 4 to 5.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-22.png)
+
+And now your customized publish page is ready for your customers:
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-20.png)
+
+## Change the Fee Address
+
+At this point, we have made a lot of changes and hopefully you’re happy with the way that your marketplace is looking. Given that you now have your own awesome photo marketplace, it’s about time we talked about monetizing it. Yup, that’s right - you will earn a commission when people buy and sell photos in your marketplace. In Ocean V4, there are a whole host of new fees and customization options that you can use (read more about that here).
+
+When someone sets the pricing for their photos in your marketplace, they are informed that a commission will be sent to the owner of the marketplace. You see that at the moment this fee is set to zero, so you’ll want to increase that. And in order to receive the fees you’ll you need to set the address that you want to recieve these fees in.
+
+![marketCustomisation](../../.gitbook/assets/market-customisation-23.png)
```

Comment:
In this photo, the dynamic pricing schema is chosen. I suggest we use the fixed price one

Reply:
Yep, done